### DISCUSSION TITLE
How to disable health verification for specific channels?

### DISCUSSION THREAD

**Discussion by PyAntony** [smallrye/smallrye-reactive-messaging#1720] [Q&A] [answered]
Hi,

I need to disable health verification for a channel:
![image](https://user-images.githubusercontent.com/24689636/165143178-66a25582-1b4d-4798-89ca-a6fe719bfeb4.png)

I am manually stopping that channel after some time, so behavior is expected. How can I remove channel from health verification after I stop it? 

By the way, I am stopping channel doing the following:
```java
    @Inject 
    KafkaClientService clientService;
    
    // calling this method somewhere else
    public void stopConsumption() {
        var consumer = (ReactiveKafkaConsumer<?, ?>) clientService.getConsumer(channelInName);
        consumer.close();
    }
```

**Comment by cescoffier** [comment, marked as answer]
You can disable the health checks for a specific channel using the `health-enabled` attribute (and set it to `false`). Note that this cannot be done dynamically.

See https://smallrye.io/smallrye-reactive-messaging/3.14.0/kafka/health/
